docs/content/integration/deployment/bare-metal.md

```diff
@@ -62,6 +62,10 @@ $ nix-env -iA nixpkgs.authelia
 In addition to the [binaries](#binaries) we publish, [FreshPorts](https://www.freshports.org/www/authelia/) offer a
 package.
 
+We publish an [rc.d](https://people.freebsd.org/~olivierd/porters-handbook/rc-scripts.html) service script file:
```

Comment:
That link is really internal. You probably want to use: https://docs.freebsd.org/en/articles/rc-scripting/

Reply:
No problem, can be replaced.